guidelines/releases-guidelines/02-kyma-release-process.md

```diff
@@ -164,7 +164,13 @@ Follow these steps to release another Kyma version.
 
 8. Validate the `yaml` and changelog files generated under [releases](https://github.com/kyma-project/kyma/releases).
 
-9. Update the release content manually with links to the instruction on how to install the latest Kyma release. Currently, this means to grab the links from the previous release and update the version number in URLs. If contributors want you to change something in the instruction, they would address you directly.
+9. Update the release content manually with links to:
+
+- Instructions on local Kyma installation
+- Instructions on cluster Kyma installation
+- Release notes
+
+For installation instructions, use the links from the previous release and update the version number in URLs. If contributors want you to change something in the instruction, they would address you directly.
```

Comment:
Maybe we should mention that the release master should cooperate with the TW who is responsible for the given release notes? 🤔